README.md

```diff
@@ -84,9 +84,9 @@ and expressive framework for building concurrent applications.
 Add the following to your `build.sbt`:
 
 ```scala
-resolvers += "JitPack" at "https://jitpack.io"
+resolvers += "jitpack" at "https://jitpack.io"
 
-libraryDependencies += "com.github.suprnation" % "cats-actors_2.13" % "1.0.0"
+libraryDependencies += "com.github.suprnation" % "cats-actors" % "1.0.2"
```

Comment:
I think this should be
```
libraryDependencies += "com.github.suprnation" %% "cats-actors" % "1.0.2"
```
(Experienced Scala devs wouldn't even blink at it, but relative newbies often stumble on this sort of thing, so I'm sensitive to it.)

Reply:
Agree. Will fix this shortly.